source/module_io/read_input_item_other.cpp

```diff
@@ -1931,6 +1931,16 @@ void ReadInput::item_others()
         read_sync_int(lr_nstates);
         this->add_item(item);
     }
+    {
+        Input_Item item("nocc");
+        item.annotation = "the number of occupied orbitals to form the 2-particle basis ( <= nelec/2)";
+        read_sync_int(nocc);
+        item.reset_value = [](const Input_Item& item, Parameter& para) {
```

Comment:
Please add Uts in read_input_item_test.cpp to ensure this code is covered in tests.
It is also better to add this parameter in read_input_ptest.cpp